rp-events/template.yaml

```diff
@@ -0,0 +1,71 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Static infrastructure for Relying Party Event Processing
+Transform: AWS::Serverless-2016-10-31
+
+Parameters:
+  Environment:
+    Description: The environment type
+    Type: String
+    AllowedValues:
+      - dev
+      - build
+      - staging
+      - integration
+      - production
+
+Conditions:
+  IsProduction: !Equals [!Ref Environment, 'production']
+  NotTestEnvironment:
+    !Or [
+      !Equals [!Ref Environment, production],
+      !Equals [!Ref Environment, integration]
+    ]
+  TestEnvironment: !Not [Condition: NotTestEnvironment]
+
+Resources:
+  ####################
+  # DynamoDB Resources
+  ####################
+
+  RelyingPartyConfigTable:
+    Type: AWS::DynamoDB::Table
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      TableName: !Sub ${AWS::StackName}-rp-config-table
+      TimeToLiveSpecification:
```

Comment:
I don't think a config table will ever want to have TTL, because it's essentially static data.

Reply:
Removed in latest commit